source-hubspot-native/source_hubspot_native/resources.py

```diff
@@ -115,14 +406,47 @@ def open(
             tombstone=Property(_meta=Property.Meta(op="d")),
         )
 
-    return Resource(
+    return common.Resource(
         name=Names.properties,
         key=["/_meta/row_id"],
         model=Property,
         open=open,
         initial_state=ResourceState(),
         initial_config=ResourceConfig(
-            name=Names.properties, interval=timedelta(days=1)
+            name=Names.properties, interval=timedelta(minutes=7)
         ),
         schema_inference=True,
     )
+
+def custom_objects(
+    cls: type[CRMObject], http: HTTPSession
+) -> common.Resource:
+    
+    def open(
```

Comment:
A comment (is that what they are called? docstrings is maybe the right term?) on this function would be nice too.

Reply:
A comment for this specific function was added here a484280098b57da301ffc5bc09ab89fa835fc8d0